internal/domain/application_oidc.go

```diff
@@ -41,24 +41,24 @@ type OIDCApp struct {
 	State AppState
 }
 
-func (h OIDCApp) GetApplicationName() string {
-	return h.AppName
+func (c *OIDCApp) GetApplicationName() string {
+	return c.AppName
 }
```

Comment:
IMO a would be a better var-name
```suggestion
func (a *OIDCApp) GetApplicationName() string {
	return a.AppName
}
```